### PR TITLE
chore(wallet) rename pokt urls to grove in default networks and tests

### PR DESCRIPTION
**Pull request by stefandunca**
Desktop PR [#13191](https://github.com/status-im/status-desktop/pull/13191)

Updates status-desktop [#13189](https://github.com/status-im/status-desktop/issues/13189)

### CODE REVIEW

**Comment by jakubgs** [comment]
Please rebase.

**Review by dlipicar — APPROVED** [review]

File: api/default_networks.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ var mainnet = params.Network{
 var goerli = params.Network{
```
Comment (dlipicar) [inline api/default_networks.go:45]: why don't we have an entry for sepolia here?

Comment (stefandunca) [reply api/default_networks.go:45]: Good point might be that we added it to the desktop only. We inject config at onboarding and login so we overwrite this. There was a discussion to drop network client injection and rely on status-go, not sure what happened with that.

Comment (stefandunca) [reply api/default_networks.go:45]: Here it is https://github.com/status-im/status-desktop/issues/11435

**Review by saledjenic — APPROVED** [review]

**Pull request merged**